joboffers/models.py

```diff
@@ -323,3 +348,16 @@ def state_label_class(self):
 
     class Meta:
         proxy = True
+
+
+class JobOfferVisualization(models.Model):
+    """
+    Model to track visualization of joboffers
+    """
+    created_at = models.DateTimeField(auto_now_add=True)
+    month_and_year = models.PositiveIntegerField()
+    event_type = models.PositiveSmallIntegerField(
+        choices=EventType.choices, verbose_name=_('Tipo de Evento')
+    )
+    session = models.CharField(max_length=40, verbose_name=_('Identificador de Sesión'))
+    joboffer = models.ForeignKey(JobOffer, on_delete=models.CASCADE)
```

Comment:
Revisemos esto. No es buena idea que "los registros de métricas" tengan claves foráneas a los "modelos de trabajo". Eso complica luego el borrado de cosas puntuales, o limpieza de tablas cuando se llenó mucho.

Una práctica habitual es guardar en "el log" la info necesaria para mostrar. ¿Qué sería eso de joboffer?

Reply:
Tengo que guardar cual es la oferta que se está visualizando, si no no la puedo distinguir para mostrar los gráficos de esa oferta. En este caso JobOfferVisualization es el hijo en la relación con el JobOoffer, borrar una visualization no afecta al modelo padre. Si podría afectar si se borra la oferta, que actualmente te va a borrar las visualizaciones, pero actualmente no se permite borrar (salvo que uses el admin).

Reply:
Ok.